package.json

```
@@ -28,17 +28,18 @@
     "**/*": "prettier --write --ignore-unknown"
   },
   "dependencies": {
-    "@sinclair/typebox": "0.32.14",
-    "ajv": "8.12.0",
     "fast-glob": "3.3.2",
+    "lodash": "^4.17.21",
```

Let's pin dependencies to a specific version 🙏🏻
```suggestion
    "lodash": "4.17.21",
```